mono/metadata/icall-def.h

```diff
@@ -364,7 +364,7 @@ NOHANDLES(ICALL(COMPINF_6, "internal_index_icall", ves_icall_System_Globalizatio
 
 ICALL_TYPE(CULDATA, "System.Globalization.CultureData", CULDATA_1)
 ICALL(CULDATA_1, "fill_culture_data", ves_icall_System_Globalization_CultureData_fill_culture_data)
-ICALL(CULDATA_2, "fill_number_data", ves_icall_System_Globalization_CultureData_fill_number_data)
+NOHANDLES(ICALL(CULDATA_2, "fill_number_data", ves_icall_System_Globalization_CultureData_fill_number_data))
```

Comment:
Maybe a comment? Revisiting after months, the `NOHANDLES` confused me, looked wrong.
It is `NOHANDLES` due to some combination of:
 1. ref formed in managed, i.e. managed frames remain conservatively scanned for the time being, and icalls do not need changing when that changes

 2. ref to value type; probably not relevant
 3. valuetype on stack
 4. valuetype contains no pointers.

Some/all of that suffices.